user/test/com/google/gwt/emultest/java11/util/function/PredicateTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.gwt.emultest.java11.util.function;
+
+import com.google.gwt.emultest.java.util.EmulTestBase;
+import java.util.function.Predicate;
+
+public class PredicateTest extends EmulTestBase {
```

Comment:
Checkstyle is failing on this class (but reviewdog isn't annotating it for you..) since this doesn't have javadoc. Add a simple javadoc comment to the class like other jre tests have and it will be happy.

```xml
<file name="/path/to/gwt/user/test/com/google/gwt/emultest/java11/util/function/PredicateTest.java">
<error line="21" severity="error" message="Missing a Javadoc comment." source="com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocTypeCheck"/>
</file>
```